scripts/src/ui/page/timed.multichoice.question.tsx

```diff
@@ -19,6 +19,8 @@ interface TimerState {
 	time: number;
 }
 
+const getImageUrl = (imageName: string) => `https://s3.amazonaws.com/htmlpoll.kinecosystem.com/images/${imageName}.svg`;
```

Comment:
Maybe extract it to a helpers/utils file and import it into all places needed, so if the URL changes in the future it'll be easy to change.

Reply:
Actually, looking at it now, we need to change the URL.